grafanalib/core.py

```diff
@@ -213,6 +213,31 @@ def DashboardLink(dashboard, uri, keepTime=True, title=None):
     }
 
 
+def Template(default, dataSource, label, name, query):
```

Comment:
Took me a while to figure out what 'default' means. Can you think of a better name? No worries if not.